docs/Example HOWTO.md

```diff
@@ -120,24 +120,17 @@ _nmos-query._tcp        PTR     qry-api-1._nmos-query._tcp
 Now we add `SRV` records that return the URL for the registration and query servers.  In this case, both of the records point to `rds1.example.com`.
 
 ```
-; NMOS RDS services                          TTL     Class  SRV  Priority  Weight  Port  Target
-reg-api-1._nmos-register._tcp.example.com.     3600    IN     SRV  10        10      80    rds1.example.com.
-qry-api-1._nmos-query._tcp.example.com.        3600    IN     SRV  10        10      80    rds1.example.com.
+; NMOS RDS and Query Services       Type   Priority  Weight   Port   Host 
+reg-api-1._nmos-register._tcp        SRV     10        10      80    rds1.example.com.
+qry-api-1._nmos-query._tcp           SRV     10        10      80    rds1.example.com.
 ```
 
 We add `TXT` records which provide information relevant to the IS-04 specification
 
 ```
 ; Additional metadata relevant to the IS-04 specification. See IS-04 specification section "Discovery: Registered Operation"
-reg-api-1._nmos-register._tcp.example.com.        TXT     "api_ver=v1.0,v1.1,v1.2,v1.3" "api_proto=http" "pri=0" "api_auth=false"
-qry-api-1._nmos-query._tcp.example.com.           TXT     "api_ver=v1.0,v1.1,v1.2,v1.3" "api_proto=http" "pri=0" "api_auth=false"
-```
-
-We add records which associate the NMOS register and NMOS query with the RDS.
-```
-; RDS                              TTL     Class  SRV  Priority  Weight  Port  Target
-_nmos-register._tcp.example.com.     3600    IN     SRV  10        20      80    rds1.example.com.
-_nmos-query._tcp.example.com.        3600    IN     SRV  10        20      80    rds1.example.com.
```

Comment:
It was specifically these lines which I think are prohibited by RFC 6763, so thanks for deleting them.

Reply:
No problem